slitlessutils/core/preprocess/crrej/group.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from slitlessutils.core.wfss import WFSSCollection
+from ....logger import LOGGER
+
+
+def group_by_visit(files, return_unique_visits=False, **kwargs):
+    """
+    Group a list of files based on their unique visits.
+
+    Parameters
+    ----------
+    files : list of str
+        List of filenames
```

Comment:
```suggestion
        List of input filenames to be grouped for cosmic ray rejection.
```
Maybe this suggestion is too specific - would this function be used for anything else?

Reply:
I'm not entirely sure. @Russell-Ryan would have a better answer for this one.

Reply:
No, I think the description is ok.  I can't think of any other use case for this code.

Reply:
Cool, ignore this suggestion then.